source/authentication/custom-jwt.txt

```diff
@@ -347,6 +347,11 @@ There are three values that you need to specify for each metadata field:
          the default name will be the last part of the notation. For example, if 
          you specify a path of ``location.primary.city``, the default value for 
          the name is ``city``.
+
+       -  If the field name contains a period (``.``) character, use a backslash 
```

Comment:
apologies for the confusion! this is actually for the Path field, not the Field Name field – since the Path value can be a path, it is parsed with dot notation, which is why we'd need to escape the period